### PR TITLE
Polkadot v0.9.37

### PR DESCRIPTION
**Pull request by pifragile**
This is the node part of https://github.com/encointer/pallets/pull/299

### CODE REVIEW

File: client/src/community_spec.rs [hunk 1]
```diff
@@ -43,18 +43,15 @@ impl CommunitySpec for serde_json::Value {
...
-			_ => (),
+		if let GeoJson::FeatureCollection(ref ctn) = geoloc {
+			for feature in &ctn.features {
+				let val = &feature.geometry.as_ref().unwrap().value;
+				if let geojson::Value::Point(pt) = val {
+					let l = Location { lon: Degree::from_num(pt[0]), lat: Degree::from_num(pt[1]) };
+					loc.push(l);
+					debug!("lon: {} lat {} => {:?}", pt[0], pt[1], l);
+				}
+			}
```
Comment (clangenb) [inline client/src/community_spec.rs:54]: Clippy is using chat gpt as a backend now I heard.

File: node/Cargo.toml [hunk 2]
```diff
@@ -76,13 +77,15 @@ encointer-balances-tx-payment-rpc-runtime-api = { git = "https://github.com/enco
...
+try-runtime-cli = { version = "0.10.0-dev", optional = true, git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.37" }
 
 
 [features]
 default = []
 runtime-benchmarks = [
-    "encointer-node-notee-runtime/runtime-benchmarks"
+    "encointer-node-notee-runtime/runtime-benchmarks",
+    "frame-benchmarking/runtime-benchmarks",
+	"frame-benchmarking-cli/runtime-benchmarks",
```
Comment (clangenb) [inline node/Cargo.toml:88]: I believe this is a whitespace vs. tab issue here.

File: runtime/Cargo.toml [hunk 3]
```diff
@@ -111,14 +111,15 @@ std = [
...
-    "frame-system-benchmarking",
-    "frame-system/runtime-benchmarks",
+    "frame-benchmarking/runtime-benchmarks",
+	"frame-support/runtime-benchmarks",
+	"frame-system-benchmarking/runtime-benchmarks",
+	"frame-system/runtime-benchmarks",
+	"pallet-balances/runtime-benchmarks",
+	"pallet-grandpa/runtime-benchmarks",
+	"pallet-timestamp/runtime-benchmarks",
+	"sp-runtime/runtime-benchmarks",
```
Comment (clangenb) [inline runtime/Cargo.toml:121]: alignment

**Review by clangenb — COMMENTED** [review]
Cool, almost nothing to add! It seems the compiler became even more powerful. :)

Anyhow, it seems that you need to rebase in order to run the CI.

File: client/src/community_spec.rs (hunk 1 above)
Comment (pifragile) [reply client/src/community_spec.rs:54]: haha

**Comment by pifragile** [comment]
@clangenb i fixed your comments. closing this in favor of https://github.com/encointer/encointer-node/pull/306

**Pull request closed without merging**